integration_tests/androidx_test/src/test/java/org/robolectric/integrationtests/axt/MissingClassInterceptorTests.java

```diff
@@ -0,0 +1,64 @@
+package org.robolectric.integrationtests.axt;
+
+import static org.junit.Assert.fail;
+
+import androidx.annotation.NonNull;
+import androidx.biometric.BiometricPrompt;
+import androidx.fragment.app.FragmentActivity;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import java.security.NoSuchAlgorithmException;
+import java.util.concurrent.Executor;
+import javax.crypto.Cipher;
+import javax.crypto.NoSuchPaddingException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.android.controller.ActivityController;
+
+/** Test intercepting classes not present in OpenJDK. */
+@RunWith(AndroidJUnit4.class)
+public class MissingClassInterceptorTests {
+
+  private FragmentActivity fragmentActivity;
+
+  @Before
+  public void setUp() {
+    fragmentActivity =
+        ActivityController.of(new FragmentActivity()).create().resume().start().get();
+  }
+
+  @Test
+  public void shouldNotCrashWithNoSuchMethodError()
```

Comment:
This test method name is very common.

Reply:
Suggestion?

Reply:
I renamed it to biometricPromptAuthenticateShouldNotCrashWithNoSuchMethodError()